config/templates/referral-reward-earned.html

```
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title></title>
```

HTMLHint (1.9.2)

[error] <title></title> must not be empty.

(title-require)